src/components/PathDay/PathDay.css

```diff
@@ -36,4 +38,21 @@
 
 .pathway-difficulty {
     margin-top: 50px;
+    text-transform: lowercase;
+}
+
+.pathway-difficulty a:link { 
+    text-decoration: none; 
+}
+
+.pathway-difficulty a:visited { 
+    text-decoration: none; 
+}
+
+.pathway-difficulty a:hover { 
+    text-decoration: none; 
```

Comment:
I think it would be fine to leave the default underline on hover, since regular search results from the main page has them!